### PR TITLE
FRLG Any% Routes

### PR DESCRIPTION
**Pull request by HunarPG**
**Please include the route in the name of the PR (for example, `AS Any% - Fix Archie 2`)**

## Route ID (for example, `alpha-sapphire-any-percent`)

frlg-any%(high exp) and frlg-any%(begginer/hiker nob) routes

## Summary of changes

Adding both routes to the base Ranger repository

## Checklist
* [ ✔] I have tested these changes and Ranger does not display any errors or warnings.
* [ ✔] All image embeds have relative paths.
* [ ✔] The route contains all fights required for the category.

### CODE REVIEW

File: frlg-any%(beginner hiker nob route)/route.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "title": "Any%(Begginer)",
```
Comment (Corvimae) [inline frlg-any%(beginner hiker nob route)/route.json]: `Any% (Beginner)`

**Review by Corvimae — CHANGES_REQUESTED** [review]
Looks mostly good. Could you fix the title of the beginner route and rename the folders to:

`frlg-any-percent-beginner` and `frlg-any-percent-high-xp` (or similar)

Worried about special characters and spaces causing unforseen issues.

Thanks! (and sorry about the slow review)

File: frlg-any%(beginner hiker nob route)/route.json (hunk 1 above)
Comment (HunarPG) [reply frlg-any%(beginner hiker nob route)/route.json]: Ok sure

Comment (HunarPG) [reply frlg-any%(beginner hiker nob route)/route.json]: Done and Pushed

File: frlg-any-percent-beginner/route.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "title": "frlg-any-percent-beginner",
```
Comment (Corvimae) [inline frlg-any-percent-beginner/route.json]: sorry, the title here should still be `Any% (Beginner)`, I just meant to change the folder name. didn't mean to be confusing 😓

File: frlg-any-percent-high-xp/route.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+    "title": "frlg-any-percent-high-xp",
```
Comment (Corvimae) [inline frlg-any-percent-high-xp/route.json]: same with this, `Any% (High XP)`

**Review by Corvimae — CHANGES_REQUESTED** [review]

File: frlg-any-percent-beginner/route.json (hunk 2 above)
Comment (Corvimae) [reply frlg-any-percent-beginner/route.json]: keep the folder name to what you changed it to that's perfect.

File: frlg-any-percent-high-xp/route.json (hunk 3 above)
Comment (HunarPG) [reply frlg-any-percent-high-xp/route.json]: Ok will do

File: frlg-any-percent-beginner/route.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+    "title": "FRLG Any% High Exp",
```
Comment (Corvimae) [inline frlg-any-percent-beginner/route.json]: These names appear to be flipped? Either that or the folder names are flipped.

**Review by Corvimae — CHANGES_REQUESTED** [review]

File: frlg-any-percent-beginner/route.json (hunk 4 above)
Comment (HunarPG) [reply frlg-any-percent-beginner/route.json]: should be fixed

**Review by Corvimae — APPROVED** [review]

**Pull request merged**